docker/Dockerfile

```diff
@@ -116,7 +115,7 @@ COPY . .
 # It creates fake empty test binaries so dependencies are built, but Zebra is not fully built.
 #
 # TODO: add --locked when cargo-chef supports it
-RUN cargo chef cook --tests --release --features "${TEST_FEATURES} ${FEATURES}" --workspace --recipe-path recipe.json
+RUN cargo chef cook --tests --release --features "${TEST_FEATURES}" --workspace --recipe-path recipe.json
```

Comment:
Will we not miss some features if we only use `$TEST_FEATURES`?

Reply:
`$TEST_FEATURES` includes `$FEATURES` from here: https://github.com/ZcashFoundation/zebra/pull/7200/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR18

Reply:
Oh, I assumed that if I used `--build-arg TEST_FEATURES="foo"`, I would override the default value, and `TEST_FEATURES` would contain only `foo`, excluding `FEATURES`. Was that wrong?

Reply:
In other words, `TEST_FEATURES` will contain `FEATURES` only if we don't set `TEST_FEATURES`. However, here https://github.com/zcashfoundation/zebra/blob/7b9a0aed04efaf61d35988283fea5d2c841da0f3/.github/workflows/build-docker-image.yml#L165 we set `TEST_FEATURES`.

Reply:
Oh, now I got it! I oversighted that use case. Let me think if the previous approach is the only way to handle this.

Reply:
Reverted: [revert(docker): revert to using extra ENTRYPOINT_FEATURES](https://github.com/ZcashFoundation/zebra/pull/7200/commits/285c67468978bb29528658afaf049e9100ff0d89)